src/mumble/Messages.cpp

```diff
@@ -1301,7 +1303,8 @@ void MainWindow::msgRequestBlob(const MumbleProto::RequestBlob &) {
 ///
 /// @param msg The message object containing the suggestions
 void MainWindow::msgSuggestConfig(const MumbleProto::SuggestConfig &msg) {
-	if (msg.has_version() && (msg.version() > Version::getRaw())) {
+	bool newVersionRequested = (msg.has_version_2() && (msg.version_2() > Version::getRaw())) || (msg.has_version() && (msg.version() > Version::toLegacyVersion(Version::getRaw())));
+	if (newVersionRequested) {
 		Global::get().l->log(Log::Warning,
 							 tr("The server requests minimum client version %1").arg(Version::toString(msg.version())));
```

Comment:
Here we can no longer simply access `msg.version`
1. We can't feed the legacy version into `Version::toString`
2. If present, we want to prefer `msg.version_2`